apps/io-services-cms-webapp/src/config.ts

```diff
@@ -27,12 +29,27 @@ export const JiraConfig = t.interface({
 });
 export type JiraConfig = t.TypeOf<typeof JiraConfig>;
 
+export type IDecodableConfigPostgreSQL = t.TypeOf<
+  typeof IDecodableConfigPostgreSQL
+>;
+export const IDecodableConfigPostgreSQL = t.interface({
+  DB_HOST: NonEmptyString,
+  DB_IDLE_TIMEOUT: withDefault(NumberFromString, 30000),
+  DB_NAME: NonEmptyString,
+  DB_PASSWORD: NonEmptyString,
+  DB_PORT: NumberFromString,
+  DB_SCHEMA: NonEmptyString,
+  DB_TABLE: NonEmptyString,
+  DB_USER: NonEmptyString,
+});
+
 // global app configuration
 export type IConfig = t.TypeOf<typeof IConfig>;
 export const IConfig = t.intersection([
   t.type({ isProduction: t.boolean }),
   InternalStorageAccount,
   JiraConfig,
+  IDecodableConfigPostgreSQL,
```

Comment:
Should we use the same type naming convention, i.e. without the leading "I"?

Reply:
honestly I agree with you. I just copied and pasted that naming from other repo 😄 
@balanza what do you think?

Reply:
Fine for me

Reply:
fixed with c5e0eb1ff8a663800955a65399d6d333d8a522e2